graphics/mesa/appdomain.te

```diff
@@ -3,3 +3,4 @@
 allow appdomain hal_graphics_allocator_default_tmpfs:file { read write map };
 allow appdomain gpu_device:dir r_dir_perms;
 allow { appdomain -isolated_app } sysfs_app_readable:file r_file_perms;
+get_prop(appdomain, vendor_video_codec_prop)
```

Comment:
you'd better put this line of rule to the dir where vendor_video_codec_prop is declared.

Reply:
Thanks Zhenlong, another PR to help test, https://github.com/projectceladon/device-intel-sepolicy/pull/125